gui/threadhandler.h

```diff
@@ -257,6 +257,7 @@ protected slots:
     int mRunningThreadCount{};
 
     bool mAnalyseWholeProgram{};
+    QString mCtuInfo;
```

Comment:
It looks like this is never set and thus always empty.

Reply:
good catch :+1: I need to investigate this